dist/sarif.xsl

```diff
@@ -40,7 +40,9 @@
     
     <xsl:template name="rules_category">
         <xsl:param name="parentTags"/>
-        <xsl:variable name="tags" saxon:assignable="yes" select="concat($qt,@desc,$qt)"/>
+        <xsl:variable name="category_desc"><xsl:call-template name="escape_illegal_chars"><xsl:with-param name="text" select="@desc" /></xsl:call-template></xsl:variable>
+        <xsl:variable name="tags" saxon:assignable="yes" select="concat($qt,$category_desc,$qt)"/>
+
```

Comment:
Is this change related to invalid command line?

Reply:
no, the xsl in dist folder was not up to date with the one in sources. I have fixed it